text/0000-ember-cli-docs.md

```diff
@@ -0,0 +1,183 @@
+running common commands like `ember serve` and who has a "zero
+config" type of experience with Ember
+2. Power users - developers who make their own configurations to the
+build pipeline
+3. Beginner addon authors - those who are looking to build simple
+shared UI components, methods, or wrappers for existing npm libraries
+4. Advanced addon authors - those who dig into internals to make their
+addon work, or who are planning for broad extensibility
+
+### Table of Contents
+
+Applying these User Personas to the CLI content, the following Table of
+Contents emerges. "Beginner" topics will include links to later "Advanced"
+topics, similar to how the Guides link to the API docs. This topic 
+arrangement may change as writing progresses, but community feedback is
+requested so no important topics are skipped.
+
+- Landing page 
+    - how to install ember cli 
+    - a very simple, short definition of what it is (the official way to create, build, and test an Ember app)
+- Overview
+    - What is the CLI
+    - Docs layout
+    - Why it should be used
+    - How to contributoe
+- Quickstart Tutorial
+    - ember new
+    - ember server
+    - ember test
+    - ember generate
+    - ember install (incl some general info about ember addons vs npm packages)
+- Basic use (explain options of each)
+    - using the "help" commmand
+    - ember new
+    - ember server
+    - ember test
+    - ember generate
+    - ember install (incl link to later section on shims for npm packages)
+    - feature flags & configurations
+    - Environmental variables
+    - File tree reference
+    - addons/dependencies
+    - Upgrading
+- Advanced use
+    - shims
+    - broccoli
+    - custom blueprints
+    - CSS compilation
+    - Using another testing library
+    - more on dependencies
+    - more configurations
+- Addon authoring
+    - Overview
+    - What is an addon (technically)
+    - Learning to build addons
+    - Creating a standalone addon
+    - Creating an in-repo addon
+    - Making an npm package wrapper
+    - Using the dummy app
+    - Including assets
+    - Testing your addon
+
+### Contributor involvement
+
+A sample app with sone written content is available [here]().
+Once the table of contents has been scaffolded, there will be a
+quest issue and call for contributors. Many sections will include
+comments and pointers for writers, so that they do not 
+need to read the entire guides to know the scope.
+Contributors will especially needed for addons and advanced
+documentation.
+
+### Versioning
+
+Only one version of the documentation will be deployed.
+The documentation app itself will have clear releases
+as major changes are made, so that users working on
+older apps can still go back in time if they need to. 
+
+### Availability of past content
+
+The old site will remain publicly available under 
+a "legacy" url, with a
+deprecation banner on each page. The new app will
+likewise include a prominent link to the old documentation.
+
```

Comment:
Should think through at what point we could swap over the old site for the new content